packages/codemirror-graphql/package.json

```diff
@@ -33,17 +33,14 @@
     "variables",
     "results"
   ],
-  "options": {
-    "mocha": "--full-trace --require resources/mochaBootload **/*-test.js"
-  },
   "scripts": {
     "build": "yarn build-clean && yarn build-js && yarn build-esm && yarn build-flow .",
     "build-js": "babel src --root-mode upward --ignore **/__tests__/**,**/__mocks__/** --out-dir .",
     "build-esm": "cross-env ESM=true babel src --root-mode upward  --ignore **/__tests__/**,**/__mocks__/** --out-dir esm && node ../../resources/renameFileExtensions.js './esm/{**,!**/__tests__/}/*.js' . .esm.js",
     "build-clean": "rimraf {mode,hint,info,jump,lint}.{js,esm.js,js.flow} && rimraf esm results utils variables coverage __tests__",
     "build-flow": "node ../../resources/buildFlow.js",
     "watch": "babel --optional runtime resources/watch.js | node",
-    "test": "nyc --require @babel/polyfill --reporter lcov --reporter text mocha $npm_package_options_mocha"
+    "test": "nyc --require @babel/polyfill --reporter lcov --reporter text jest"
```

Comment:
Just doesn't need istanbul at all. it comes already there. so if you just do --coverage when running tests it will get you the reports you need.

Reply:
yes! also no need for `@babel/polyfill` now either, since we are using `jest-ts`

Reply:
an example of this can be found [here](https://github.com/gagoar/ts-node-template/blob/main/jest.config.js)

Reply:
in this case, the `codemirror-graphql` `jest.config.js` should look the same as all the others, where it just resolves the shared config which takes care of this. we're using the `projects` option in the root config. like other packages, this PR should provide a single line for `jest.config.js` and should only execute `jest` for `test` script. all the jest configuration is already taken care of for the whole monorepo. may need to tweak things a bit for the codemirror mock though